vtu11/impl/parallel_helper_impl.hpp

```diff
@@ -0,0 +1,69 @@
+﻿//          __        ____ ____
+// ___  ___/  |_ __ _/_   /_   |
+// \  \/ /\   __\  |  \   ||   |
+//  \   /  |  | |  |  /   ||   |
+//   \_/   |__| |____/|___||___|
+//
+//  License: BSD License ; see LICENSE
+//
+#ifndef VTU11_PARALLEL_HELPER_IMPL_HPP
+#define VTU11_PARALLEL_HELPER_IMPL_HPP
+
+#include "vtu11_impl.hpp"
+namespace vtu11
+{
+  template<typename Writer>
+  void writePVTUfile( const std::string& path,
+                      std::string baseName,
+                      const std::vector<DataSet>& pointData,
+                      const std::vector<DataSet>& cellData,
+                      size_t numberOfFiles,
+                      Writer writer)
+  {
+    std::string parallelName = path + baseName + ".pvtu";
+    std::ofstream output( parallelName, std::ios::binary );    
+    size_t ghostLevel = 0;//Hardcoded to be 0
+    std::vector<double> points;
```

Comment:
what does this do?
Maybe @phmkopp knows better?

Reply:
The points vector? I don't think its used anywhere?

Reply:
We needed an empty std::vector<double> as input parameter for calling the function detail::addDataSet(). It is probably not a very elegant way to do this, but we had a lack of other ideas there. ;-)

Reply:
Well the problem is that you are not adding a data set. You just want to write the tag. So I guess the better way of doing it is somehow separating the logic for writing data sets and xml tags.

Reply:
I think this is probably one of the things Philipp and I can think about once we have merged.

Reply:
If this is not even worse coding style, we could also create the "attributes" directly within this "PPoints-Scope" and then just write an empty tag...
So instead of the line

_detail::addDataSet( writer, output, points, 3, "", true );_

we could write the following:

_StringStringMap attributes = { { "type", dataTypeString<double>() }, { "NumberOfComponents", std::to_string(3) } };
writer.addDataAttributes(attributes);
writeEmptyTag(output, "PDataArray", attributes);_

Reply:
Do we only every need those arguments in the pvtu file, even if in the vtu files we have additional arguments for example for offsets, etc?

Reply:
In the paper about vtk file formats, there are also only the point data and cell data array types and the point types stored in the pvtu. https://vtk.org/wp-content/uploads/2015/04/file-formats.pdf
![image](https://user-images.githubusercontent.com/57192391/108369383-d3424e00-71fb-11eb-879d-ba167c2d76b7.png)

And the offset, celltypes and the connectivity are all only kind of indices, anyway.